\graphicspath{{chapters/_resources/}}

\chapter{Transcription}

\hypertarget{transcription-initiation}{%
\section{Transcription initiation}\label{transcription-initiation}}

RNA polymerases are~\emph{highly processive enzymes}, able to continuously extend transcripts without releasing them, yet unable to resume transcription if they are detached from the nascent RNA.

\textbf{RNA pol II} is responsible for the transcription of all protein-coding genes, plus snoRNA genes, miRNA genes, siRNA genes, lncRNA genes and most snRNA genes. RNA pol II enzyme is composed of 12 subunits.

The \textbf{core promoter}~is a short sequence encompassing \textasciitilde{} 50bp upstream and \textasciitilde{} 50bp downstreram of the~TSS~consisting of core promoter elements.

We start from the unbound promoter and \textbf{TFIID complex}, formed by TAFs (14-15 subunits) and TBP. \textbf{TBP} is responsible for the recognition of the TATA box, TAF1-2 for Ini region and TAF6-9 for MTE and DPE.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-16_at_12-11-22.png}
\caption{Sainsbury et al., Nature Review Mol. Cell. Biol. 2015}
\end{figure}

Once TFIIA and TFIIB reach TFIID, the upstream promoter complex forms. In particular, the promoter region becomes bent. Lastly, Pol II and TFIIF are recruited at the promoter region to initiate transcription.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-16_at_15-53-19.png}
\caption{Sainsbury et al., Nature Review Mol. Cell. Biol. 2015}
\end{figure}

Once ATP is released, promoter clearance is activated allowing the PIC complex to reach the open conformation and to start transcription. \textbf{TFIIH} induces torsional stress by rotating with respect to a fixed site (TBP bound), melting DNA leading to the open complex state (OC). TFIIH promotes unwinding of approximately 10 bp of promoter DNA. When Pol II locates the TSS, it transits into a processive elongation mode. Dissociation of general transcription factors enables the formation of the Pol II elongation complex (addition of elongation factors).

Summarising, transcription initiation follows:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  TFIID + TBP promoter recognition $\rightarrow$ PIC assembly
\item
  TFIIA recruitment stabilises the TBP-DNA complex
\item
  TFIIB stabilises TBP-DNA and TBP-DNA-Pol II complexes, binds the template strand to position DNA for initiation of RNA syntehsis (setting the TSS); stimulates RNA synthesis by allosterically rearranging active site residues and stabilizing closed polymerase clamp
\item
  TFIIF heterodimer associates with polII and prevents non-specific interaction of polII with DNA; stabilizes the PIC by interacting also with TFIIB; influences TSS selection; stimulates phosphodiester bonds formation
\item
  TFIIE binds pol II (clamp) and facilitates the recruitment of TFIIH to the initiation complex; provides a bridge between polII and TFIIH; binds single stranded DNA; TFIIE stabilizes the open promoter; facilitates ATPase/kinase activities of TFIIH
\item
  TFIIH opens promoter DNA, phosphorilates RNA pol II CTD; promotes translocation of ssDNA into polII cleft
\item
  Factors binding modify the structural conformation of the preassembled complex TFIID-TBP-promoter structure becomes visible to TFIIB and A, the hole structure is visible to RNAPol II-TFIIF which is a compelx and so on\ldots{}
\item
  Pol II is positioned at the core promoter by the combination of TFIID, TFIIA and TFIIB through protein-protein interactions and protein-DNA interactions
\item
  TFIIH then melts 10-15 bp of DNA in order to position the single strand template in the polII cleft (open complex) to initiate RNA synthesis
\item
  The CTD of PolII will be phosphorilated by the TFIIH kinase subunit during the first 30 bp of transcription when PolII loses its contacts with GTFs before proceeding onto the elongation stage
\end{enumerate}

How does this start?~\emph{What makes a promoter all of a sudden accessible to TBP and so to TFIID?}~ The DNA sequence is always there, what makes it visible/accessible to the general transcription factors?

\hypertarget{chromatin-remodeling}{%
\section{Chromatin remodeling}\label{chromatin-remodeling}}

Nucleosomes are the functional units of protein. In nucleosomes the position of each major groove facing the histone octamer is designed as a \textbf{superhelix location} (SHL), numbered from 0 at the Dyad to ±7.

Proteins cannot easily associate with DNA sequences that touch the nucleosomal histone surface. Nucleosomal DNA is bent around the histone octamer~--~target sequences can be distorted or unrecognizable. GC-rich sequences with AA, TT or TA dinucleotides spaces by 10 bp bend more easily and display higher affinity for the histone octamer~``nucleosome positioning sequences''.

Nucleosome unwrapping is influenced by histone PTMs. In particular, certain combinations of PTMs display synergistic effects. For instance, covalent modifications of histone proteins can significantly alter the organization and function of chromatin. Examples:

\begin{itemize}
\tightlist
\item
  acetylated H3K56 enhances the unwrapping of the DNA at the entry/exit sites of the nucleosome.
\item
  acetylated H4K91 leads to nucleosome instability - H4K91 lays in the H3-H4 and H2A-H2B interaction surface
\end{itemize}

Histone modifications may also recruit proteins and enzyme to regulate the chromatin state. Remember that nucleosomal fibers can fold into higher order structures that may be even less accessible.

With the use of energy nucleosomes can be remodelled:
\begin{itemize}
\tightlist
\item
Histone octamers can slide along DNA
\item
Can be fully or partially disassembled
\item
Histones can be replaced with histone variants and post-translationally modified
\end{itemize}

These processes are carried out by the synergic activity of \textbf{chromatin remodeling complexes} and \textbf{histone chaperones}.

\hypertarget{techniques-to-study-chromatin-remodelling}{%
\subsubsection{Techniques to study chromatin remodelling}\label{techniques-to-study-chromatin-remodelling}}

\begin{itemize}
\tightlist
\item
  \textbf{MNase:} micrococcal nuclease is able to digest linker DNA, therefore to isolate nucleosomes. We have no information about the sequence, but we can assess the bp length of isolated DNA. Cons: some mapped entities are not nucleosomal, preference for cutting A/T rich regions.
\item
  \textbf{ChIP-seq}: after DNA sonication, immunoprecipitation and immunocomplex purification are performed (with protein-specific antibodies). DNA and proteins are cross-linked and purified, then bound DNA is analyzed by massively parallel short-read sequencing. ChIP-seq is relatively simple (technically), but does not have a high resolution due to non-uniform DNA fragmentation. Resolution also depends on library/sequencing setting.
\end{itemize}

\hypertarget{nucleosome-organization}{%
\section{Nucleosome organization}\label{nucleosome-organization}}

Nucleosome organization can be described as a combination of nucleosome occupancy and positioning:

\begin{itemize}
\tightlist
\item
  \emph{Occupancy}: average number of nucleosomes measured within a specific genomic region within a cell population
\item
  \emph{Positioning}: probability of a nucleosome reference point being at a specific genomic coordinate (same dyad)
\end{itemize}

Most nucleosomes in animal genomes are poorly positioned, but there are regions with \textbf{phased arrays} of well positioned nucleosomes (Figure \ref{fig:nucleosome}) .

\begin{figure}
\centering
\includegraphics[width=0.7\textwidth]{../_resources/Screenshot_2022-09-16_at_16-26-04.png}
\caption{Nucleosome occupancy}
\label{fig:nucleosome}
\end{figure}

Most promoters contain a nucleosome free region (NFR) and a defined nucleosomal architecture that aids the recruitment of Pol II. A subset of genes may have a nucleosome in their promoter region, which is depleted during gene activation. DNA sequences containing relatively high GC content tend to wrap nucleosomes with higher affinity in vitro.

ATP-dependent chromatin remodellers act at promoter regions. \textbf{Remodels the structure of chromatin (RSC)} is a chromatin remodelling factor acting on core promoters. It retains a cavity which can accomodate nucleosomes.

\hypertarget{nucleosome-remodelling}{%
\subsection{Nucleosome remodelling}\label{nucleosome-remodelling}}

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-16_at_16-30-29.png}
\caption{Becker and Workman, Cold Spring Harbor Laboratory, 2013}
\label{fig:peeling}
\end{figure}

The unpeeling of DNA segments from the histone surface can lead to a delocalization of nucleosomes (their sliding), removal or replacement of histones or to the complete eviction of the nucleosome, also dependent on associated histone chaperones (Figure \ref{fig:peeling}).

ATP-dependent chromatin remodelling enzymes can act cooperatively with \textbf{histone chaperones} to generate or disassemble nucleosomes. Chaperones associate with histones upon their synthesis, escort them into the nucleus, and aid in their specific association with DNA during different processes such as DNA replication, repair, or transcription. Histone Chaperones participate in assembly or disassembly nucleosomes in vitro or in vivo without using the energy of ATP.

Example: SWR and Inositol-requiring protein 80 (INO80) chromatin remodelling complexes regulate histone turnover and histone variant H2A.Z deposition. HIRA complex regulates H3.3 deposition at genes and regulatory elements.

\textbf{H2A.Z} has extended acidic patches on the surface which stimulate remodeling activities and weaken DNA interaction and may play a role in Pol II recruitment - it is lost at promoters upon RNA PolII loading. H2A.Z comprises 15\% of total H2A and it is post translationally modified (acetyated, polyubiquitinilate\ldots).

The most rapid \textbf{nucleosome turnover} occurs over promoters, tRNA, and small nucleolar RNA genes (often studied in over expression settings).

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-23_at_15-22-44.png}
\caption{Venkatesh and Workman, Nature Rev Mol Cell Biol, 2015}
\label{fig:pronucleo}
\end{figure}

PIC formation at a core promoter requires a nucleosome depleted region (Figure \ref{fig:pronucleo}). This could be achieved by \textbf{RSC} binding to the AT-rich region in the promoter, freeing the NDR. Next, \textbf{HIRA} performs the replacement of H3-H4 by H3.3-H4 and \textbf{SWR} replaces the H2A-H2B dimer by H2A.Z-H2B dimer. Finally, \textbf{INO80} replaces H2A.Z-H2B by H2-A and H2B. The resulting scenario exhibits unmethylated CpG islands in NDR region, allowing for PIC formation.

\subsection{Chromatin Immunoprecipitation} 
(ChIP) experiments to determine protein-DNA interactions require a \emph{crosslinking} step.

\begin{enumerate}
\tightlist
\item
  Crosslinking: formaldehyde and a quencher are required to covalently modify DNA

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-22_at_21-36-39.png}
\caption{Cross linking}
\end{figure}

\item
  Lysis
\item
  Antibody binding
\item
  Immunoprecipitation
\item
  Wash steps
\item
  Crosslink reversal
\item
  DNA purification and quantitative PCR
\item
  DNA and protein analysis
\end{enumerate}
 Chromatin Immunoprecipitation coupled to sequencing analyses can be used to determine protein-DNA interactions.
 
\hypertarget{main-nucleosome-modifications-in-promoters}{%
\subsection{Main nucleosome modifications in promoters:}\label{main-nucleosome-modifications-in-promoters}}

Most promoters (98\%) occupied by Pol II are also occupied by histone H3K4me3 and acetylated H3K9 and H3K14 (Gunter et al., 2007).

\begin{itemize}
\tightlist
\item
  \textbf{H3K4me3}-modified nucleosomes are found at sites of transcription. H3K4me3 enrichment is detectable in almost 80\% of all protein-coding genes within 1 kb of known or predicted transcript start sites. Only 30-40\% of protein coding genes are expressed in human embryonic stem cells. Histone H3K4me3 is observed also at the promoters of genes for which there is no evidence of transcription in ES cells - typically lower than in active genes. Genes not enriched in H3K4me3 are found in clusters along the genome; The vicinity of these genes on the genome suggest that their expression they may be regulated sinergically.
\item
  \textbf{H3K9-14Ac} were enriched at the promoters of nearly 70\% of genes, including both transcriptionally active and inactive genes. Nearly all of the promoters (\textgreater95\%) acetylated on H3K9 and H3K14 were also enriched for H3K4me3.
\item
  \textbf{H3K36me3} and \textbf{H3K79me2} modifications occur almost exclusively downstream of promoters that produce detectable transcripts in ES cells
\end{itemize}

\hypertarget{rt-qpcr-based-detection-of-5-rna-transcripts-of-inactive-genes}{%
\subsubsection{RT-qPCR based detection of 5' RNA transcripts of inactive genes}\label{rt-qpcr-based-detection-of-5-rna-transcripts-of-inactive-genes}}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Assay components and DNA template: forward primer + probe + reverse primer
\item
  Denatured template and annealing assay components:
\item
  Polymerization and signal generation
\end{enumerate}

Genes that do not produce full length mRNAs can experience transcription initiation. The majority of all genes contain H3K4me3-modified nucleosomes in both ES and differentiated cells. Differential H3K4 methylation is found in 25\% of genes. These genes have cell type-specific expression pattern and cell type-specific function.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-22_at_21-49-05.png}
\caption{Guenther et al., Cell 2007}
\end{figure}

\textbf{Summary:} Results suggest that most protein-coding genes in human cells, including most genes thought to be transcriptionally inactive, experience the hallmarks of transcription initiation. H3K4me3 and H3K9-14Ac modifications, together with RNA Pol II, occupy the promoters of approximately 79\% of protein-coding genes in ES and differentiated cells, but only about half of these produce detectable transcripts. The co-occupancy of Pol II with H3K4me3- and H3K9,14Ac-modified nucleosomes at genes without detectable levels of transcription suggests that a large fraction of human genes experience transcription initiation without transcript completion.

Protein-coding genes can fall into three groups of regulatory behaviour:

\begin{itemize}
\tightlist
\item
  \emph{Productive}: the actively transcribed genes are occupied by nucleosomes with histone modifications that are hallmarks of both initiation (H3K4me3 and H3K9-14Ac) and elongation (H3K36me3 and H3K79me2)
\item
  \emph{Non-productive}: experiences transcription initiation without evidence of transcript elongation or accumulation.
\item
  \emph{No initiation}: genes that are excluded from experiencing transcription initiation, where mechanisms that prevent transcription initiation must predominate.
\end{itemize}

\hypertarget{global-run-on-sequencing-gro-seq}{%
\subsubsection{Global run-on sequencing (GRO-seq)}\label{global-run-on-sequencing-gro-seq}}

Global run-on sequencing (GRO-Seq) enables to map the presence of elongating RNA pol in the chromatin context by detection of the nascent transcripts. The \emph{Sarkosyl treatment} permeabilizes nuclear membranes, prevents RNA Pol from binding DNA (if not already bound), disassembles chromatin and dissociates pausing factors.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-22_at_21-58-03.png}
\caption{Core et al, 2008}
\end{figure}

Results: only few paused genes are transcriptionally inactive.
\textbf{Pausing} can be seen as a mechanism for tuning expression from active genes rather than a means of gene inactivation. The paused Pol II remains stably associated with the nascent RNA and is fully capable of resuming elongation; however, further signals are needed to elicit the transition to a productive elongation complex.

\emph{Pausing index}: ratio of Pol II density on promoter to gene body.

The \textbf{CTD} undergoes dynamic changes in phosphorylation and dephosphorylation during transcription. The CTD is a long unstructured domain consisting of \(Y_1S_2P_3T_4S_5P_6S_7\) tandem repeats. \textbf{Cyclin-dependent kinases} (CDKs) are serine/threonine specific kinases; cell cycle-associated CDKs regulate cell cycle progression. Transcription-associated CDKs are key regulators of gene expression. PTMs of the PolII CTD coordinate transcription and RNA processing

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-22_at_22-09-19.png}
\caption{Differential phosphorylation of RNA Pol II CTD during transcription.}
\label{fig:phCTD}
\end{figure}

The differential phosphorylation of RNA Pol II CTD during transcription is reported in Figure \ref{fig:phCTD}.

\hypertarget{transcription-pausing-mechanism}{%
\subsubsection{Transcription pausing mechanism}\label{transcription-pausing-mechanism}}

\textbf{NELF} is recruited on PolII by \textbf{DSIF}, which contributes to pausing by forming 2 nucleic acid clamps. CDK9 with cyclin T, referred to as \textbf{P-TEFb} complex, enables pause-release leading to transcription elongation (Figure \ref{fig:dsif}) .

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-09-22_at_22-14-15.png}
\caption{Jonkers \& Lis, Nature Rev Mol Cell Biol, 2015}
\label{fig:dsif}
\end{figure}

Pol II distribution profiles surrounding all promoters reveal a general decrease in promoter occupancy upon NELF depletion. Knockdown of NELF and consequent reduced Pol II pausing decreases gene expression, because nucleosomes occupy the promoter regions and thus block the expression of these genes by preventing TF and GTF binding.

\textbf{Promoter-proximal pausing} represents an additional layer of regulation to accommodate increased demands for precise and rapid gene regulation during organism development and responses to stress. It might be beneficial to maintain highly regulated promoters poised in an open chromatin state, to prevent their incorporation into the more inaccessible, condensed heterochromatin that exists in metazoans. Housekeeping genes lack pronounced pol II pausing.

Main hypotheses for the function of pol II pausing:

\begin{itemize}
\tightlist
\item
  establishing permissive chromatin: paused Pol II helps to maintain the nucleosome-deprived structure by blocking nucleosome assembly over promoter sequences. Pausing would thus keep the promoter region accessible for activator and transcription factor binding.
\item
  rapid or synchronous activation: at a gene with paused Pol II, gene activation could proceed simply through recruitment of positive transcription elongation factor b (P-TEFb), thereby triggering the rapid release of paused Pol II into productive elongation without the need of the PIC assembly.
\item
  integration of signals: pausing represents a separate step in the transcription cycle for factors to act and allows for combinatorial control between transcription factors that recruit the transcription machinery (TF1) and those that trigger pause release (TF2), where both would be necessary for gene activation.
\item
  checkpoint in early elongation: on the left, arrows depict interactions between the capping enzyme complex (CEC) and DSIF--NELF as well as the Ser5 phosphorylation of the carboxy-terminal heptapeptide repeat domain (CTD) of Pol II, which is thought to stimulate capping activity. The hat represents the 5 RNA cap. In the centre, P-TEFb-dependent phosphorylation events release paused Pol II and create a platform for binding of RNA-processing factors (RPFs) on the Ser2-phosphorylated CTD of Pol II, as shown on the right.
\end{itemize}

7SK snRNP represses transcription by sequestering and inhibiting P-TEFb. Super elongation complex (\textbf{SEC}) and Bromodomain containing protein 4 (\textbf{BRD4}) complexes represent active forms of P-TEFb that can promote the release of Pol II from pausing.

\textbf{BRD4} competes for binding to P-TEFb with the P-TEFb inhibitory complex HEXIM-- 7SK, and it is recruited to TSSs by multiple means, including histone acetylation. The SEC can interact with a subset of co-activators such as Mediator, polymerase- associated factor 1 (PAF1) and Integrator, the latter of which is a complex that interacts with the CTD of Pol II. The SEC and BRD4 predominantly mediate the recruitment P-TEFb, but their regulatory importance and composition with regard to paused Pol II release seems to vary across different genes, cell types and stimuli. Ser2 phosphorylated CTD promotes the recruitment of other transcription elongation factors, including SPT6.

A large repertoire of transcription activators crucially involved in cancer, including MYC and NF-kB, target P-TEFb, DSIF or NELF to gene promoters.

\section{Transcription elongation}

During the elongation phase, RNA pol II faces nucleosomes. In order to proceed with elongation, RNA pol II needs to overcome the issue: \emph{histone turnover} is predominantly observed at transcription start sites.

\hypertarget{structural-basis-of-the-nucleosome-transition-during-rna-pol-ii-passage---kujirai-et-al-2018}{%
\subsection{Structural basis of the nucleosome transition during RNA pol II passage - Kujirai et al 2018}\label{structural-basis-of-the-nucleosome-transition-during-rna-pol-ii-passage---kujirai-et-al-2018}}

The aim of the research paper was to reconstitute RNA pol II and a nucleosome separately in vitro and analyze their interaction. The DNA stretch is required to be around 153 bp to achieve histone wrapping for obtaining a nucleosome. Keep in mind that is not easy to recapitulate the PIC formation in vitro. The fist step involves unwinding and exposing DNA. For bypassing this step, it is possible to use an already ``melted'' initiation site ready for transcription. Once RNA pol II and \textbf{TFIIS} elongation factor are added, transcription can start.

\hypertarget{nucleosome-structure}{%
\subsubsection{Nucleosome structure}\label{nucleosome-structure}}

A single base pair is centred on the nucleosome \emph{dyad}, which defines the pseudo 2-fold symmetry axis of the nucleosome. The major groups in the DNA stretch facing the dyad are \emph{superhelical location} (SHL) -7 and -1. In nucleosomes, SHLs are formed at regular intervals (10bp) from the dyad interacting with histone K and R residues.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_22-41-07.png}
\caption{Kujirai and Kurumizaka, \emph{Curr Op. Struct. Biol.,} 2020}
\end{figure}

Kujirai and Kurumizaka, \emph{Curr Op. Struct. Biol.,} 2020

\hypertarget{cryo-em}{%
\subsubsection{Cryo-EM}\label{cryo-em}}

Cryo-Electron Microscopy is technique applied for molecule structure determination, which allows to reconstruct structures almost at the atomic level. We freeze samples in vitro (-80° C, -90° C) with liquid ethane. The sample is hit with a photon beam instead of light from different orientations, obtaining a pool of 2D images. Finally, a 3D image is reconstructed through an algorithm, providing a resolution in Armstrongs.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_22-57-06.png}
\caption{Screenshot 2022-10-05 at 22.57.06.png}
\end{figure}

Most of the times, RNA Pol II was associated with the nucleosome at position SHL(-5) or SHL(-1), as well as SHL(-6) and SHL(-2). Other sites were quickly encountered and passed by Pol II, whereas the aforementioned ones required a bit more time. Looking at the electrophoresis results without the addition of TFIIS, RNA pol II can only reach SHL(-5), while when we add it also SHL(-1) is observed.

RNA Pol II elongates RNA up to SHL(-5) in the absence of the elongation factor and until reaching SHL(-1) in the presence of TFIIS. Pausing at SHL (-5) and (-1) represent rate limiting steps of nucleosomal transcription in vitro.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_22-54-39.png}
\caption{Kujirai at al., \emph{Science,} 2018}
\end{figure}

Kujirai at al., \emph{Science,} 2018

SHL(-6) is at the entry site, DNA has not been ``peeled'' yet from the nucleosome. At these sites H3 and H4 interact with DNA, we can appreciate the presence of modifications.

The dimer H2A and H2B is completely exposed after the ``peeling'' of the first DNA turn. This results in the uncovering of a positively charged region, which can be easily accessed by other compounds and rewrapped by ``foreign DNA'' i.e.~not the DNA which was there previously.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_23-02-04.png}
\caption{Screenshot 2022-10-05 at 23.02.04.png}
\end{figure}

\hypertarget{summary-of-the-main-results}{%
\subsubsection{Summary of the main results}\label{summary-of-the-main-results}}

\begin{itemize}
\tightlist
\item
  Nucleosomes can inhibit initiation and elongation of transcription.
\item
  Nucleosome free regions are present within core promoters.
\item
  Transcription pausing occurs at the first well-positioned nucleosome (+1).
\item
  RNA polII transiently pauses when it transcribes nucleosomal DNA.
\item
  Elongation factors are required for productive transcription.
\item
  The transcription efficiency is modulated by histone post-translational modifications, histone variants and histone exchange which may affect histone-DNA contacts in the nucleosome releaving the major RNAPII pausing sites.
\end{itemize}

\hypertarget{template-looping}{%
\subsubsection{Template looping}\label{template-looping}}

DNA upstream of Pol II starts wrapping around the upstream nucleosome, pushing pol II to continue. This process does not require disassembly.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_23-04-42.png}
\caption{Kujirai and Kurumizaka., \emph{Curr Op. Struct. Biol.,} 2020
Template looping results in histone transfer from ahead of the transcribing RNApol II to behind}
\end{figure}

Kujirai and Kurumizaka., \emph{Curr Op. Struct. Biol.,} 2020
Template looping results in histone transfer from ahead of the transcribing RNApol II to behind

It is not only required to have TFIIS, other factors are involved: Spt4/5 and Paf1C (elongation factors), \textbf{FACT} and \textbf{Spt6} (histone chaperones) with acidic patches balancing the positively charged histones exposed in unwrapping.

\textbf{FACT}(facilitate chromatin transcription) interacts with the H2A-H2B dimer and we observe the formation of the \emph{hexasome} (only 8 subunits) in the process of histone turnover. FACT activity is also influenced by H2A-H2B dimers ubiquitylation. Nap1 (HC) stabilizes hexasomes favouring transcription elongation.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_23-05-01.png}
\caption{Lai and Pugh, \emph{Nature Review Mol Cell Biol}. 2017}
\end{figure}

Lai and Pugh, \emph{Nature Review Mol Cell Biol}. 2017

In a similar fashion, H3.3 is bound by HIRA, TF for ``recycling'', regulated by acetylation.

\textbf{Paf1C} elongation factor interacts directly with Pol II, in particular with s5 phosphorylation. It also interacts with DSIF and FACT. When pol II is elongating, it promotes the phosphorylation of NELF and DSIF. Ctr9 is a subunit of Paf.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_23-05-58.png}
\caption{Van Oss \emph{et al., TRENDS Biochem Sci.} 2017}
\end{figure}

Van Oss \emph{et al., TRENDS Biochem Sci.} 2017

\hypertarget{structural-basis-of-nucleosome-disassembly-and-reassembly-by-rna-p-ii-elongation-complex-with-fact---ehara-et-al.}{%
\subsubsection{Structural basis of nucleosome disassembly and reassembly by RNA P II elongation complex with FACT - Ehara et al.}\label{structural-basis-of-nucleosome-disassembly-and-reassembly-by-rna-p-ii-elongation-complex-with-fact---ehara-et-al.}}

Recent article: once H2A-H2B are partially exposed, FACT starts interacting with the nucleosome. When the full exposure of the nucleosome is reached, FACT promotes a transit from downstream to upstream, assisting the template looping procedure.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-10_at_10-35-12.png}
\caption{Screenshot 2022-10-10 at 10.35.12.png}
\end{figure}

Chromatin remodeling factors also partecipate in transcription elongation:

\begin{itemize}
\tightlist
\item
  \textbf{RSC}: arrangement of NDR between +1 and -1 nucleosomes
\item
  \textbf{CHD1} and \textbf{ISWI}: arrangement of nucleosome spacing and separation of closely packed nucleosomes
\item
  \textbf{Fun30}: facilitation of nucleosome disassembly
\end{itemize}

\hypertarget{termination}{%
\section{Transcription termination}\label{termination}}

Transcription termination occurs when the elongation complex is disassembled, and both RNA Pol II and the RNA are released. In our cells a signal on DNA is required, a \emph{polyA site} followed by U rich/G rich regions. Once polyA is transcribed, we observe the SPSF complex recruitment, which recognizes the AAA site and proceeds to RNA cleavage and polyadenylation. Meanwhile, RNA pol II will continue transcription (slowly, probably due to allosteric changes in pol II structure bound to the complex) of mRNA which will be degraded due to the exposed 5' end. \textbf{\emph{Torpedo model:}} XRN2 degrades this RNA and leads to the dissociation of pol II from DNA.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-10_at_10-38-48.png}
\caption{Porrua \& Libri, Nature Review Mol Cell Biol, 2015}
\end{figure}

Porrua \& Libri, Nature Review Mol Cell Biol, 2015

\hypertarget{promoter-reporter-assay}{%
\subsection{Promoter reporter assay}\label{promoter-reporter-assay}}

Since transcription at core promoters can be induced by distal elements, we can perform a promoter reporter assay to investigate which regions actively regulate transcription.

In order to perform the assay, we need to clone the upstream region of a known gene e.g.~GFB. \emph{Cloning vector}: enhancer + CNV (promoter and enhancer)+ multiple cloning sites (\textbf{MCS}) + GFB (coding sequence + polyA site) + OriC + amp.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Linearize a plasmid by cutting it with restriction enzyme (HIND) and run it on the gel for extraction
\item
  Ligation
\item
  Transformation through heat shock
\item
  Integration screening (check amp resistance)
\end{enumerate}

After cloning, we transfect the compound in cells. GFB will be barely detectable. We could start again with a longer sequence and clone it in from of GFB, but it would still be undetectable. It is required to employ a longer gene for observing signal. By playing with deletions, we understand that we only need small regions: 100 bp at TSS and a proximal enhancer region.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-10_at_10-45-54.png}
\caption{Anderson \& Sandelin, \emph{Nature Review Gen}. 2019}
\end{figure}

Anderson \& Sandelin, \emph{Nature Review Gen}. 2019

The enhancer can be hundreds of kilobases away from the promoter, could be upstream or downstream, even in introns. In order to check if a gene is an enhancer we need to place it around a reporter gene with different combinations.

\begin{itemize}
\tightlist
\item
  \emph{Core promoter}: sequence that enables to initiate transcription (immediate vicinity of a TSS) by docking the pre-initiation complex (PIC)
\item
  \emph{Promoter}: refers to a sequence which can autonomously drive high levels of productive transcription (proximal enhancer + core promoters)
\item
  \emph{Enhancer}: distal element. The enhancer can be transcribed, but the mRNA is not stable and not codifying for protein.
\end{itemize}

\hypertarget{promoter-types}{%
\subsection{Promoter types}\label{promoter-types}}

Not all promoters have a TATA box, we have 3 different kinds of promoters:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \textbf{Type I ('adult')}: TATA box and sharp TSS. No CpG, nucleosomes are not well positioned $\rightarrow$ minority of promoters. Tissue-specific expression.
\item
  \textbf{Type II ('ubiquitous')}: no TATA box and CpG islands. Broad TSS, ordered nucleosome configuration.
\item
  \textbf{Type III ('developmentally regulated'):} featured by polycomb regulation, large CpG islands extending into the body of gene.
\end{enumerate}

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-05_at_10-05-59.png}
\caption{Lenhard \emph{et al., Nature Rev Cancer} 2012}
\end{figure}

Lenhard \emph{et al., Nature Rev Cancer} 2012

\hypertarget{transcription-factor-binding}{%
\subsection{Transcription factor binding}\label{transcription-factor-binding}}

Most of the times, TFs require co-factors and direct binding; sometimes the \emph{effector domain} can be either a ligand-binding domain or PPI domain.

TFs cooperate and synergize to gain access to their binding sites. For instance, factor A can bind alone to the unwinded first part, then B binds. In general we can observe:

\begin{itemize}
\tightlist
\item
  passive cooperativity
\item
  TF-TF interactions
\item
  enhanceosome model
\end{itemize}

Most TF binding occurs in clusters (\textless1kb) on nucleosome depleted regions. Regulatory networks are usually created with the same TF regulating different genes depending on other TF and cofactors availability.

\textbf{Pioneer TFs} are able to access their own binding sites independently of nucleosome/chromatin binding. For instance, \textbf{FoxA} HTH domain is really similar to H1 linker histone. HTH binds DNA minor groove: it has a trans activator domain and it is able to bind core histone. Once pioneer TF bind, they are able to pull DNA from the nucleosome creating a more accessible stretch. Examples:

\begin{itemize}
\tightlist
\item
  SOX2 exposes DNA for OCT4 $\rightarrow$ \emph{sequential binding mechanism} for recruiting multiple TFs\emph{,} rather than simultaneous
\item
  FoxA1 enables Estrogen Receptor alpha recruitment
\end{itemize}

Other mechanisms we can observe are the recruitment of acetylases e.g.~p300 to enhance nucleosome destabilization.

Transcription is dynamically regulated but not constantly occurring. The core promoter will accomodate a number of pol II, which will then transcribe (burst). Different core promoters can have a different burst size. Enhancers do not change this size (which is given by promoter structure), but increase the number of bursts per time.

\begin{figure}
\centering
\includegraphics[width=0.5\textwidth]{../_resources/Screenshot_2022-10-10_at_10-50-43.png}
\caption{Haberle \& Stark, \emph{NatRevMolCelBiol.} 2018}
\end{figure}


